src/Entity/Storage/AppStorage.php

```diff
@@ -94,6 +94,31 @@ public function loadUnchanged($id) {
     return parent::loadUnchanged($id);
   }
 
+  /**
+   * Load app by UUID.
+   *
+   * This function is more efficient than loadUnchanged(),
+   * because it does not need to cover the case
```

Comment:
Every line should be 80 characters long max, you can still add some text to this one. Please fix it.

Reply:
fixed